flink-metrics/flink-metrics-prometheus/src/test/java/org/apache/flink/metrics/prometheus/PrometheusReporterTest.java

```diff
@@ -70,9 +69,25 @@
 	@Rule
 	public ExpectedException thrown = ExpectedException.none();
 
-	private final MetricRegistry registry = new MetricRegistry(MetricRegistryConfiguration.fromConfiguration(createConfigWithOneReporter("test1", "" + NON_DEFAULT_PORT)));
-	private final FrontMetricGroup<TaskManagerMetricGroup> metricGroup = new FrontMetricGroup<>(0, new TaskManagerMetricGroup(registry, HOST_NAME, TASK_MANAGER));
-	private final MetricReporter reporter = registry.getReporters().get(0);
+	private MetricRegistry registry;
+	private FrontMetricGroup<TaskManagerMetricGroup> metricGroup;
+	private PrometheusReporter reporter;
+	private int port;
```

Comment:
do we need this extra field for that? Can't we access `reporter.getPort()`? Isn't it redundant?